users/views.py

```diff
@@ -67,4 +69,44 @@ def post(self, request):
             return JsonResponse({'message':'LOGIN_SUCCESS','token':access_token,'user_name':user.nickname}, status=200)
 
         except KeyError:
-            return JsonResponse({'message':'KEY_ERROR'}, status=400)
\ No newline at end of file
+            return JsonResponse({'message':'KEY_ERROR'}, status=400)
+    
+class LikeView(View):
+    @login_decorator
+    def post(self, request):
+        try:
+            data = json.loads(request.body)
+            product_id = data['product_id']
```

Comment:
```suggestion
            data       = json.loads(request.body)
            product_id = data['product_id']
            user       = request.user
```
- =을 기준으로 정렬해 주세요.
- request.user를 밑에서 자주 사용하므로 변수에 담아서 사용해도 좋을 거 같아요

Reply:
= 기준으로 정렬 했습니다.
user = request.user를 사용했습니다.